src/Data/Vec/Base.agda

```diff
@@ -358,3 +356,12 @@ last xs with initLast xs
 transpose : Vec (Vec A n) m → Vec (Vec A m) n
 transpose []         = replicate []
 transpose (as ∷ ass) = replicate _∷_ ⊛ as ⊛ transpose ass
+
+
+------------------------------------------------------------------------
+-- DEPRECATED
+------------------------------------------------------------------------
+-- Please use the new names as continuing support for the old names is
+-- not guaranteed.
+
+-- Version 2.0
```

Comment:
This deprecated section is empty? Was it supposed to contain the `join` that was removed above?

Reply:
Hmmm. It previously did do so, until I realised that that definition had only been introduced recently enough for it not to require deprecation... (see the discussion on the issues above). But then I lost my nerve, and left this section in, as a placeholder... 'just in case'. My belief is that it, too, could be removed. Should it be?

Reply:
Yup, that's fine. If it was introduced but never released, it can be zapped without a warning.